x/protocolpool/keeper/keeper.go

```diff
@@ -48,6 +49,7 @@ func NewKeeper(cdc codec.BinaryCodec, storeService storetypes.KVStoreService,
 		cdc:            cdc,
 		authority:      authority,
 		BudgetProposal: collections.NewMap(sb, types.BudgetKey, "budget", sdk.AccAddressKey, codec.CollValue[types.Budget](cdc)),
+		ContinuousFund: collections.NewMap(sb, types.ContinuousFundKey, "continuous_fund", sdk.AccAddressKey, codec.CollValue[types.ContinuousFund](cdc)),
```

Comment:
should these be private as well?

Reply:
I don't think so. We are using them in tests for setting and getting values.